Tools/check-ci-validation-opt-out.sh

```
@@ -0,0 +1,63 @@
+
+for file in "${targets[@]}"; do
+    line_number=0
+    while IFS= read -r line || [[ -n "$line" ]]; do
+        line_number=$((line_number + 1))
+        trimmed="${line#"${line%%[![:space:]]*}"}"
+        if [[ "$trimmed" == \#* ]]; then
+            continue
+        fi
+
+        if [[ "$line" =~ $pattern ]]; then
+            display_file="${file#$ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)